### PR TITLE
Document generate path property

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/1802

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[66d476a](https://github.com/NuGet/docs.microsoft.com-nuget/commits/66d476a609bdbcf76b473bb614820f902348f3d3)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-generatePathProperty/docs/consume-packages/Package-References-in-Project-Files.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1918) | [Details](#user-content-99d3622d-77e864a9)

<a id="99d3622d-77e864a9"></a>
### [docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-generatePathProperty/docs/consume-packages/Package-References-in-Project-Files.md)
 - **Line 202**: **[Warning]** `````Cannot find bookmark '#Restore-outputs' in 'reference/msbuild-targets.md', did you mean '#restore-outputs'?`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281846387010-1918%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=InoDQ0OjLYq%2FRmDdSpNcHd65%2BQQrKUVD1TDwUCGMF0w%3D&st=2020-02-28T18%3A42%3A18Z&se=2020-03-30T18%3A47%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[9a8170f](https://github.com/NuGet/docs.microsoft.com-nuget/commits/9a8170f03d8100ad64a328fe93cbf8a302f0198f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-generatePathProperty/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1918) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281848180645-1918%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=h9ohi5xDN1i976NRxclTJm97xj%2BmdmfrwhpIuBG01NI%3D&st=2020-02-28T18%3A43%3A57Z&se=2020-03-30T18%3A48%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Package-References-in-Project-Files.md [hunk 1]
```diff
@@ -165,7 +165,44 @@ Conditions can also be applied at the `ItemGroup` level and will apply to all ch
 </ItemGroup>
 ```
 
+## GeneratePathProperty
+
+*Version **4.9** or above*
```
Comment (heng-liu) [inline docs/consume-packages/Package-References-in-Project-Files.md]: As we didn't insert into 15.9, mentioned in this issue: https://github.com/NuGet/Home/issues/8374#issuecomment-515607739 , we didn't support it for 15.x.
Do we need to change it to `*This feature is available with NuGet **4.9** or above and with Visual Studio 2019 **16.0** or above.*` ?

File: docs/consume-packages/Package-References-in-Project-Files.md [hunk 2]
```diff
@@ -165,7 +165,44 @@ Conditions can also be applied at the `ItemGroup` level and will apply to all ch
...
+
+Not all packages are authored in conventional manner. Very often authors would include tools in packages, and the usage story for those packages is unclear.
```
Comment (zivkan) [inline docs/consume-packages/Package-References-in-Project-Files.md]:
```suggestion
Sometimes it is desirable to reference files in a package from an MSBuild target.
```

File: docs/consume-packages/Package-References-in-Project-Files.md (hunk 1 above)
Comment (nkolev92) [reply docs/consume-packages/Package-References-in-Project-Files.md]: Nice catch.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[f2b5823](https://github.com/NuGet/docs.microsoft.com-nuget/commits/f2b5823461577d221dacceae288f7c24f829f97d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-generatePathProperty/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1918) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002282118011860-1918%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=k5AnaeASftZgqA%2BR1OPO6jYzgvOyMzlgU0JVVmYk%2BkQ%3D&st=2020-02-28T21%3A13%3A38Z&se=2020-03-30T21%3A18%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[4e32425](https://github.com/NuGet/docs.microsoft.com-nuget/commits/4e324257199a13b1a4421fb4411094e7fec8d142)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-generatePathProperty/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1918) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002282118438530-1918%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3kGb2kalH1mo7diZdLbdWCmlJQh0IUgrZFc%2FYS%2BSw%2Fo%3D&st=2020-02-28T21%3A14%3A21Z&se=2020-03-30T21%3A19%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[88f4190](https://github.com/NuGet/docs.microsoft.com-nuget/commits/88f419007e0a7ac958b5e8cf15446547916f6e85)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-generatePathProperty/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1918) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002282119269765-1918%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=QP2%2Fdq7lf1T%2Fii%2B1o4xbz%2Bd5zkIaLrMtDT6clJKUn%2Fs%3D&st=2020-02-28T21%3A15%3A06Z&se=2020-03-30T21%3A20%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Package-References-in-Project-Files.md [hunk 3]
```diff
@@ -165,7 +165,44 @@ Conditions can also be applied at the `ItemGroup` level and will apply to all ch
...
 ```
 
+## GeneratePathProperty
+
+This feature is available with NuGet **5.0** or above and with Visual Studio 2019 **16.0** or above.*
+
+Sometimes it is desirable to reference files in a package from an MSBuild target.
+In `packages.config` based projects, the packages are installed in a folder relative to the project file. However in PackageReference, the packages are [consumed](../concepts/package-installation-process.md) from the *global-packages* folder, which can vary from machine to machine.
+
+To bridge that gap, NuGet has introduce property that points to the location from which the package will be consumed.
```
Comment (zkat) [inline docs/consume-packages/Package-References-in-Project-Files.md]:
```suggestion
To bridge that gap, NuGet introduced a property that points to the location from which the package will be consumed.
```

Comment (nkolev92) [reply docs/consume-packages/Package-References-in-Project-Files.md]: committed :)

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[2af07f4](https://github.com/NuGet/docs.microsoft.com-nuget/commits/2af07f4775d2a6cfe640b27a5a794f381ed69531)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-generatePathProperty/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1918) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002282227011275-1918%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=FdZ%2BMhDVql2NkxVnZrTJLUXqeesdDRZkqAXI08NbXiA%3D&st=2020-02-28T22%3A22%3A39Z&se=2020-03-30T22%3A27%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**